### DISCUSSION TITLE
Welcome to microG unofficial installer Discussions!

### DISCUSSION THREAD

**Discussion by ale5000-git** [micro5k/microg-unofficial-installer#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheLastGimbus** [comment]
Hi there! I'm @TheLastGimbus, fellow android-hacking lover by all means.

I use Lineage for quite a few years and very happy with it. Also using microG for >1 year because i wanted to get rid of spying and not-really-knowing-what's-going-in-my-phone, and also wanted to squeeze out last few drops of juice out of my old phones

I first used it from official Lineage4microG, and the experience was nice, but the guy (and updates) vanished for few months (luckly they returned)

I also installed it "manually" (magisk -> riru -> xposed -> sig spoofing), but found it very buggy, so I recently started to compile lineage4microg myself

I'm really looking into your installer, because it would be really awesome if i could just flash some single .zip just like everyone does with gapps

**Reply by ale5000-git** [reply]
Hi,
sorry for the late reply but I'm really busy with many things.
The main point of the installer is to easily install microG on every possible device so I'm happy if it helps.
If you have any problem with the installer just tell me :-)
